server/issue.go

```diff
@@ -23,10 +23,21 @@ import (
 	"github.com/mattermost/mattermost-plugin-jira/server/utils/types"
 )
 
-func makePost(userId, channelId, message string) *model.Post {
+const (
+	commaSeparator   = ", "
+	labelsField      = "labels"
+	statusField      = "status"
+	reporterField    = "reporter"
+	priorityField    = "priority"
+	descriptionField = "description"
+	resolutionField  = "resolution"
+)
+
+func makePost(userID,
+	channelID, message string) *model.Post {
```

Comment:
Nit pick: The formatting look weird. Should this be
```suggestion
func makePost(userID, channelID, message string) *model.Post {
```